components/docs-chef-io/content/habitat/sup_guide.md

```diff
@@ -12,111 +12,134 @@ gh_repo = "habitat"
     weight = 120
 +++
 
```

Comment:
I actually think this entire file should be removed. I don't think we have followed this process since 2018 and it is completely unrelated to the supervisor. This is likely only going to confuse and scare people. I'm scared having looked at it.

Reply:
I AGREE!

Reply:
What I saw when I read it:
![tenor-253386806](https://user-images.githubusercontent.com/4400151/115074968-40e2c180-9eaf-11eb-8056-fa5aa8c5c9f8.gif)